polyfill/lib/ecmascript.mjs

```diff
@@ -2278,12 +2096,12 @@ export function TemporalMonthDayToString(monthDay, showCalendar = 'auto') {
 }
 
 export function TemporalYearMonthToString(yearMonth, showCalendar = 'auto') {
-  const year = ISOYearString(GetSlot(yearMonth, ISO_YEAR));
-  const month = ISODateTimePartString(GetSlot(yearMonth, ISO_MONTH));
+  const year = ISOYearString(GetSlot(yearMonth, ISO_DATE).year);
```

Comment:
I couldn't add a suggestion here, but seems like a good spot to use the same `*String` pattern used above.

Reply:
Same